spec/orders.json

```diff
@@ -468,15 +611,45 @@
                 "example": "Standard Flat Rate Shipping"
               },
               "code": {
-                "type": "number",
-                "format": "float",
+                "type": "string",
                 "description": "Code that indicates the shipping type.",
-                "example": 1
+                "example": "0"
               },
               "merchant_name": {
                 "type": "string",
                 "description": "Merchant name.",
                 "example": "Virgin Club"
+              },
```

Comment:
@KameronSturgess Seems this wasn't added in the docs yet after adding it to the API